### ISSUE TITLE
Failed to spawn command /usr/bin/scss-lint. Make sure /usr/bin/scss-lint is installed and on your PATH

### ISSUE THREAD

**Issue by brentonstrine** [AtomLinter/linter-scss-lint#131]
Whenever I open an scss file i get the path error. but it is definitely in my path!

```
echo $PATH
/usr/bin/scss-lint:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/bin/scss-lint
```

**Comment by Arcanemagus** [comment]
What do you get for `which scss-lint`? Are you on OS X?

Also, unless `scss-lint` is installed in a _folder_ located at `/usr/bin/scss-lint`, then you have your PATH set wrong.

**Comment by brentonstrine** [comment]
I'm on OSX. `which scss-lint` has no response.

**Comment by Arcanemagus** [comment]
If `which scss-lint` shows no results, then it isn't installed in your PATH. How are you installing it?

**Comment by brentonstrine** [comment]
In Atom, I have installed `linter` and `linter-scss-lint`.

**Comment by Arcanemagus** [comment]
Closing this as it seems you have (possible) resolved your issue from here and have filed #132.

**Comment by brentonstrine** [comment]
Yes, thanks for closing. I fixed it -- needed to update the executable path to the path I actually installed it in (since I didn't have permissions to `/usr/bin`.
